service/history/timer_queue_active_task_executor.go

```diff
@@ -574,6 +574,7 @@ func (t *timerQueueActiveTaskExecutor) executeActivityRetryTimerTask(
 		ScheduleToStartTimeout: durationpb.New(scheduleToStartTimeout),
 		Clock:                  vclock.NewVectorClock(t.shardContext.GetClusterMetadata().GetClusterID(), t.shardContext.GetShardID(), task.TaskID),
 		VersionDirective:       directive,
+		Stamp:                  task.Stamp,
```

Comment:
we need the same thing in transfer activity task.

Reply:
Yes.

Reply:
func (t *transferQueueTaskExecutorBase) pushActivity(
(and all usagfes of AddActivityTaskRequest)

Reply:
done